test/evergreen/code_coverage/per_test_code_coverage.py

```diff
@@ -0,0 +1,261 @@
+            env = os.environ.copy()
+            if verbose:
+                print(f'env: {env}')
+                print(f'gcovr_command: {gcovr_command}')
+            try:
+                completed_process = subprocess.run(split_command, env=env, check=True)
+                output = completed_process.stdout
+                print(f'Command returned {output}')
+            except subprocess.CalledProcessError as exception:
+                print(f'Command {exception.cmd} failed with error {exception.returncode} "{exception.output}"')
+            if verbose:
+                print(f'Completed a run of gcovr on {build_copy_name}')
+    print(f"Ending run_gcovr({build_dir_base}, {gcovr_dir})")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-c', '--config_path', required=True, help='Path to the json config file')
+    parser.add_argument('-b', '--build_dir_base', required=True, help='Base name for the build directories')
+    parser.add_argument('-j', '--parallel', default=1, type=int, help='How many tests to run in parallel')
+    parser.add_argument('-g', '--gcovr_dir', help='Directory to store gcovr output in')
+    parser.add_argument('-s', '--setup', action="store_true",
+                        help='Perform setup actions from the config in each build directory')
+    parser.add_argument('-v', '--verbose', action="store_true", help='Be verbose')
+    args = parser.parse_args()
+
+    verbose = args.verbose
+    config_path = args.config_path
+    build_dir_base = args.build_dir_base
+    gcovr_dir = args.gcovr_dir
+    parallel_tests = args.parallel
+    setup = args.setup
+
+    if verbose:
+        print('Per-Test Code Coverage')
+        print('======================')
+        print('Configuration:')
+        print(f'  Config file                      {config_path}')
+        print(f'  Base name for build directories: {build_dir_base}')
+        print(f'  Number of parallel tests:        {parallel_tests}')
+        print(f'  Perform setup actions:           {setup}')
+        print(f'  Gcovr output directory:          {gcovr_dir}')
+        print()
+
+    if parallel_tests < 1:
+        sys.exit("Number of parallel tests must be >= 1")
+
+    # Load test config json file
+    with open(config_path) as json_file:
+        config = json.load(json_file)
+
+    if verbose:
+        print('  Configuration:')
+        print(config)
+
+    if len(config['test_tasks']) < 1:
+        sys.exit("No test tasks")
+
+    setup_actions = config['setup_actions']
+
+    if setup and len(setup_actions) < 1:
+        sys.exit("No setup actions")
+
+    if verbose:
+        print('  Setup actions: {}'.format(setup_actions))
+
+    if gcovr_dir:
+        if not Path(gcovr_dir).is_absolute():
+            sys.exit("gcovr_dir must be an absolute path")
+
+    build_dirs = check_build_dirs(build_dir_base=build_dir_base, parallel=parallel_tests, setup=setup, verbose=verbose)
```

Comment:
there is no need for the equal sign, as we can just pass in the argument itself.

Reply:
In this case, you're correct so I've removed them. But very useful sometime.